integration-test/src/main/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -423,4 +447,8 @@ private void deleteTables_ShouldDeleteTablesWithCoordinator() throws Exception {
   private int executeWithArgs(List<String> args) {
     return SchemaLoader.mainInternal(args.toArray(new String[0]));
   }
+
+  protected void waitForKeyspacesTableCreation() {
```

Comment:
I know it's only for Cassandra for now, but I don't think we should use the Cassandra term `keyspaces` here? Maybe we should use `namespaces` instead?

```suggestion
  protected void waitForNamespacesTableCreation() {
```

Reply:
Good point, thank you. Let me update it.